### PR TITLE
added manpage

### PR DESCRIPTION
**Pull request by shibumi**
Heyho,
I created a manpage for your project.
When you edit the PKGBUILD just make sure to gzip the manpage and move it to "${pkgdir}/usr/share/man/man1/" 

Best regards

Chris 

EDIT: nvm that upgradable is a valid english word. I have reset the other changes and just added the man page.

### CODE REVIEW

**Comment by anthraxx** [comment]
no reason to gzim anything, just move the file there and it will be gzipped per magic :8ball:

**Comment by shibumi** [comment]
Via man-db.service?

**Comment by anthraxx** [comment]
so via makepkg ```/usr/share/makepkg/tidy/zipman.sh```, nothing you need to do

File: doc/arch-audit.1 [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+.RS 4
+Show only vulnerable package names and their versions.
+.RE
+.PP 
+\fI-u, --upgradable\fR 
+.RS 4
+Show only packages that have already been fixed.
+.RE
+.PP 
+\fI-V, --versionI\fR
```
Comment (ilpianista) [inline doc/arch-audit.1]: Typo? (version**I**)

Comment (shibumi) [inline doc/arch-audit.1]: Ah yes. Thats a typo. Thanks! I will fix this when I am at home.

**Comment by ilpianista** [comment]
It's a shame that rust-lang/cargo#2729 hasn't been implemented yet.

**Pull request merged**